frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/ERDContent.tsx

```diff
@@ -58,33 +58,43 @@ export const ERDContent: FC<Props> = ({
       setEdges((edges) =>
         edges.map((e) =>
           e.source === id || e.target === id
-            ? { ...e, animated: true, data: { ...e.data, isHovered: true } }
+            ? { ...e, animated: true, data: { ...e.data, isHighlighted: true } }
```

Comment:
Rename `isHighlighted` instead of `isHovered` because nodes and edges might not hovered when flag is true.

Reply:
Thank you! I will make changes to the code based on this PR. 🙆‍♀️

Reply:
Great, I'm really thank you 🙏